shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -949,6 +950,17 @@ public void onFlutterUiNoLongerDisplayed() {
     // no-op
   }
 
```

Comment:
can you hook up the builder in this class and the appropriate defaults? i.e. how does the user supply these flags?

Reply:
Will do.